### PR TITLE
ad-server: change default domain to testdom.sink.test

### PR DESCRIPTION
**Pull request by obnoxxx**
Fixes: #39

Signed-off-by: Michael Adam <obnox@redhat.com>

### CODE REVIEW

**Review by phlogistonjohn — APPROVED** [review]
Looks good, thanks!

File: images/ad-server/provision.sh [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 set -e
 
-REALM="ZZZ-BEST.X.ASYNCHRONO.US"
-DOM="ZZZ-BEST"
+REALM="samba1.sink.test"
```
Comment (phlogistonjohn) [inline images/ad-server/provision.sh:5]: Unfortunately, I think using lower case here causes an error with the provisioning tool. You'd naturally think the domain is case sensitive but this [samba wiki page](https://wiki.samba.org/index.php/Setting_up_Samba_as_an_Active_Directory_Domain_Controller) says it must be upper cased.

**Review by phlogistonjohn — CHANGES_REQUESTED** [review]

File: images/ad-server/provision.sh (hunk 1 above)
Comment (obnoxxx) [reply images/ad-server/provision.sh]: This was not the error. It worked before with `testdom`. The problem was that `samba1` is also the netbios name of the domain controller. Updated...

**Comment by phlogistonjohn** [comment]
Sorry if I seem overly picky about this, but if you ended up changing the dc name in the end, what's wrong with samba1 now for the domain name?

**Comment by phlogistonjohn** [comment]
> > Sorry if I seem overly picky about this, but if you ended up changing the dc name in the end, what's wrong with samba1 now for the domain name?
> 
> As I may have explained (maybe elsewhere), I am usually a fan of giving test objects names that indicate what kind of object this is. I.e something with `domain` for a domain, and something with `dc` for a dc ...
> 
> You are right, that it is not needed here. If you prefer, I can remove that part and put it into a separate PR for discussion.

It's fine to leave it. I was just a bit confused about why both values ended up getting changed.

**Review by phlogistonjohn — APPROVED** [review]

**Pull request merged**